modules/liveIntentIdSystem.js

```diff
@@ -119,10 +129,20 @@ function initializeLiveConnect(configParams) {
   return liveConnect;
 }
 
+const setEventFiredFlag = function(_) {
+  eventFired = true
+}
+
 function tryFireEvent() {
   if (!eventFired && liveConnect) {
-    liveConnect.fire();
-    eventFired = true;
+    const eventDelay = liveConnect.config.fireEventDelay || 500
+    setTimeout(() => {
+      const instances = window.liQ_instances
+      instances.forEach(i => i.eventBus.once(EVENTS_TOPIC, setEventFiredFlag))
+      if (!eventFired && liveConnect) {
+        liveConnect.fire();
+      }
+    }, eventDelay)
```

Comment:
This prevents duplicate event in case apart from LiveIntent's Prebid user id module another LiveConnect wrapper is installed in the same page. Mid term the goal is to remove this guard and handle all events on the backend.